sdk/python/cmd/pulumi-language-python/main.go

```diff
@@ -381,7 +381,12 @@ func (host *pythonLanguageHost) Pack(ctx context.Context, req *pulumirpc.PackReq
 		}
 	}
 
-	buildCmd, err := tc.ModuleCommand(ctx, "build", "--wheel", "--outdir", tmp)
+	args := []string{"--wheel", "--outdir", tmp}
+	if err == nil {
```

Comment:
I think err could be the errors from pip setup here. Better to make an explicit `hasUV` flag

Reply:
Ah yes, you're right.  Fixed.